### PR TITLE
PTX-13378: Fixing logic to fetch the telemetry status

### PR DESCRIPTION
**Pull request by aghodke1312**
https://jenkins.pwx.dev.purestorage.com/job/Torpedo/view/Operator%20Release%20Cloud%20Dashboard/job/tp-nextpx-gke-op-ms/

### CODE REVIEW

File: tests/basic/telemetry_test.go [hunk 1]
```diff
@@ -271,6 +273,8 @@ var _ = Describe("{ProfileOnlyDiags}", func() {
 						return
 					}
 				})
+				// This sleep is required because the heap and stack logs might not have been written at same time.
+				time.Sleep(30* time.Second)
```
Comment (nikolaypopov) [inline tests/basic/telemetry_test.go:277]: I would do retry here instead of hardcoded sleep

File: tests/basic/telemetry_test.go [hunk 2]
```diff
@@ -55,9 +55,14 @@ func runPxctlCommand(pxctlCmd string, n node.Node, cmdConnectionOpts *node.Conne
 
 func TelemetryEnabled(currNode node.Node) bool {
 	// This returns true if telemetry is enabled
-	out, err := runPxctlCommand("status -j | jq .telemetrystatus.connection_status.error_code", currNode, nil)
-	Expect(err).NotTo(HaveOccurred(), "Error getting telemetry status for %s", currNode.Name)
-	return strings.TrimSpace(out) == TelemetryEnabledStatus
+	output, err := runPxctlCommand("status | egrep ^Telemetry:", currNode, nil)
+	Expect(err).NotTo(HaveOccurred(), "Failed to get status for nodee %v", currNode.Name)
```
Comment (nikolaypopov) [inline tests/basic/telemetry_test.go:59]: nit: nodee

File: tests/basic/telemetry_test.go (hunk 1 above)
Comment (jrivera-px) [reply tests/basic/telemetry_test.go:277]: The sleep can actually be removed.  The next step does an "Eventually" which retries the check for the newly generated stack/heap files so no need to have this sleep here at all.

**Review by jrivera-px — APPROVED** [review]
LGTM... just remove the sleep.   Thx

**Review by CharudathGopal — APPROVED** [review]

**Comment by aghodke1312** [comment]
@jrivera-px Lets say if heap file is generated and .stack is not, in that case the file list will be 1 stack file from previous run and will cause the test to fail, I can keep a 10 sec sleep so that we dont run into this condition?

**Pull request merged**